Shaders/Common/private/ComputeBlueNoiseTexture.fx

```diff
@@ -31,15 +36,24 @@ float SampleRandomNumber(uint2 PixelCoord, uint SampleIndex, uint SampleDimensio
     return (float(Value) + 0.5f) / 256.0f;
 }
 
-float2 SampleRandomVector2D(uint2 PixelCoord)
+// Roberts R1 sequence see - https://extremelearning.com.au/unreasonable-effectiveness-of-quasirandom-sequences/
+float4 SampleRandomVector2D2D(uint2 PixelCoord)
 {
-    return float2(
-        fmod(SampleRandomNumber(PixelCoord, 0u, 0u) + float(g_SSRAttribs.FrameIndex & 0xFFu) * M_GOLDEN_RATIO, 1.0),
-        fmod(SampleRandomNumber(PixelCoord, 0u, 1u) + float(g_SSRAttribs.FrameIndex & 0xFFu) * M_GOLDEN_RATIO, 1.0));
+    float G = 1.61803398875f; 
+    float Alpha = 0.5 + rcp(G) * float(g_CameraAttibs.iFrameIndex & 0xFFu);
+    return float4(
+        frac(SampleRandomNumber(PixelCoord, 0u, 0u) + Alpha),
+        frac(SampleRandomNumber(PixelCoord, 0u, 1u) + Alpha),
+        frac(SampleRandomNumber(PixelCoord, 0u, 2u) + Alpha),
+        frac(SampleRandomNumber(PixelCoord, 0u, 3u) + Alpha)
+    );
 }
 
-
-float2 ComputeBlueNoiseTexturePS(in FullScreenTriangleVSOutput VSOut) : SV_Target
+PSOutput ComputeBlueNoiseTexturePS(in float4 Position : SV_Position) 
```

Comment:
Why don't you want to use `FullScreenTriangleVS`?

Reply:
Yes, this is an error. It occurred when taking the latest changes